Test failure with parallel_fit_dask (v7.1.0rc1)

### Description

```
_____________________________________ TestFitInfo.test_all _____________________________________

self = <astropy.modeling.tests.test_fitting_parallel.TestFitInfo object at 0x75e99a28b9d0>
tmp_path = PosixPath('/tmp/sconseil/pytest-of-sconseil/pytest-14/test_all1')

    def test_all(self, tmp_path):
        fitter = TRFLSQFitter()
    
        parallel_fit_dask(
            data=self.data,
            model=self.model,
            fitter=fitter,
            fitting_axes=0,
            scheduler="synchronous",
            fit_info=True,
        )
    
        assert "message" in fitter.fit_info.properties
    
        assert_allclose(fitter.fit_info.get_property_as_array("nfev"), [0, 9, 9])
    
        param_cov_array = fitter.fit_info.get_property_as_array("param_cov")
        assert param_cov_array.shape == (3, 3, 3)
        assert_allclose(param_cov_array[0], 0)
        assert_allclose(param_cov_array[1], param_cov_array[2])
>       assert np.all(np.abs(param_cov_array[1]) > 0)
E       AssertionError: assert np.False_
E        +  where np.False_ = <function all at 0x75ea06e40df0>(array([[5.62519719e-28, 0.00000000e+00, 1.85390220e-28],\n       [0.00000000e+00, 1.89140184e-28, 4.19975573e-44],\n       [1.85390220e-28, 4.19975573e-44, 1.85769512e-28]]) > 0)
E        +    where <function all at 0x75ea06e40df0> = np.all
E        +    and   array([[5.62519719e-28, 0.00000000e+00, 1.85390220e-28],\n       [0.00000000e+00, 1.89140184e-28, 4.19975573e-44],\n       [1.85390220e-28, 4.19975573e-44, 1.85769512e-28]]) = <ufunc 'absolute'>(array([[ 5.62519719e-28,  0.00000000e+00, -1.85390220e-28],\n       [ 0.00000000e+00,  1.89140184e-28,  4.19975573e-44],\n       [-1.85390220e-28,  4.19975573e-44,  1.85769512e-28]]))
E        +      where <ufunc 'absolute'> = np.abs

../.pyenv/versions/tmp/lib/python3.13/site-packages/astropy/modeling/tests/test_fitting_parallel.py:1156: AssertionError
```

### Expected behavior

_No response_

### How to Reproduce

1. Get package from '...'
2. Then run '...'
3. An error occurs.

```python
python -m pytest --pyargs astropy.modeling.tests.test_fitting_parallel
```


### Versions

```python
import astropy
astropy.system_info()
```
```
platform
--------
platform.platform() = 'Linux-6.14.6-arch1-1-x86_64-with-glibc2.41'
platform.version() = '#1 SMP PREEMPT_DYNAMIC Fri, 09 May 2025 17:36:18 +0000'
platform.python_version() = '3.13.1'

packages
--------
astropy              7.1.0rc1
numpy                2.2.5
scipy                1.15.3
matplotlib           3.10.3
pandas               2.2.3
pyerfa               2.0.1.5
```

--- a/astropy/modeling/tests/test_fitting_parallel.py
+++ b/astropy/modeling/tests/test_fitting_parallel.py
@@ -1153,7 +1153,7 @@ class TestFitInfo:
         assert param_cov_array.shape == (3, 3, 3)
         assert_allclose(param_cov_array[0], 0)
         assert_allclose(param_cov_array[1], param_cov_array[2])
-        assert np.all(np.abs(param_cov_array[1]) > 0)
+        assert np.any(np.abs(param_cov_array[1]) > 0)
 
         # Test slicing that returns an array
 

